src/notebooks/controllers/notebookControllerManager.ts

```diff
@@ -254,12 +254,14 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
         const cachedConnections = await this.listKernels(cancelToken, 'useCache');
         const nonCachedConnectionsPromise = this.listKernels(cancelToken, 'ignoreCache');
 
+        traceVerbose(`Found ${cachedConnections.length} cached controllers`);
         // Now create or update the actual controllers from our connections. Do this for the cached connections
         // so they show up quicker.
         this.createNotebookControllers(cachedConnections);
 
         // Do the same thing again but with non cached
         const nonCachedConnections = await nonCachedConnectionsPromise;
+        traceVerbose(`Found ${cachedConnections.length} non-cached controllers`);
```

Comment:
Adding these as there seems to be a weird problem with the web extension. It doesn't log any errors but doesn't return any kernels.